### PR TITLE
[2/2] RNN 10.9 proofread full

### PR DESCRIPTION
**Pull request by goldmermaid**

### CODE REVIEW

**Comment by goldmermaid** [comment]
Please merge this PR after #541.

File: chapter_recurrent-neural-networks/lstm.md [hunk 1]
```diff
@@ -15,22 +14,21 @@ cell (as opposed to reading any other cell). We will refer to this as the
...
 design is the same as before, namely to be able to decide when to remember and
-when to ignore inputs into the latent state via a dedicated mechanism. Let us see
-how this works in practice.
+when to ignore inputs in the latent state via a dedicated mechanism. Let us see
+how LSTM works in practice.
 
 ## Gated Memory Cells
 
-Three gates are introduced in LSTMs: the input gate, the forget gate, and the output gate. In addition to that we introduce memory cells that take the same shape as the hidden state. Strictly speaking this is just a fancy version of a hidden state, custom engineered to record additional information.
+Three gates are introduced in LSTMs: the input gate, the forget gate, and the output gate. In addition to that we will introduce the memory cell that have the same shape as the hidden state. Strictly speaking this is just a fancy version of a hidden state, customized engineered to record additional information.
```
Comment (astonzhang) [inline chapter_recurrent-neural-networks/lstm.md]: cell that has?

File: chapter_recurrent-neural-networks/lstm.md [hunk 2]
```diff
@@ -40,45 +38,50 @@ $$
 \end{aligned}
 $$
 
-$\mathbf{W}_{xi}, \mathbf{W}_{xf}, \mathbf{W}_{xo} \in \mathbb{R}^{d \times h}$ and $\mathbf{W}_{hi}, \mathbf{W}_{hf}, \mathbf{W}_{ho} \in \mathbb{R}^{h \times h}$ are weight parameters and $\mathbf{b}_i, \mathbf{b}_f, \mathbf{b}_o \in \mathbb{R}^{1 \times h}$ are bias parameters.
+where $\mathbf{W}_{xi}, \mathbf{W}_{xf}, \mathbf{W}_{xo} \in \mathbb{R}^{d \times h}$ and $\mathbf{W}_{hi}, \mathbf{W}_{hf}, \mathbf{W}_{ho} \in \mathbb{R}^{h \times h}$ are the weight parameters and $\mathbf{b}_i, \mathbf{b}_f, \mathbf{b}_o \in \mathbb{R}^{1 \times h}$ are bias parameters.
```
Comment (astonzhang) [inline chapter_recurrent-neural-networks/lstm.md]: we should rm the here
even if you add the for weight parameters, why not for bias parameters?

File: chapter_recurrent-neural-networks/lstm.md [hunk 3]
```diff
@@ -40,45 +38,50 @@ $$
...
+Next we design the memory cell. Since we have not specified the action of the various gates yet, we first introduce the *candidate* memory cell $\tilde{\mathbf{C}}_t \in \mathbb{R}^{n \times h}$. Its computation is similar to the three gates described above, but it uses a $\tanh$ function with a value range for $(-1, 1)$ as the activation function. This leads to the following equation at time step $t$.
 
 $$\tilde{\mathbf{C}}_t = \text{tanh}(\mathbf{X}_t \mathbf{W}_{xc} + \mathbf{H}_{t-1} \mathbf{W}_{hc} + \mathbf{b}_c)$$
 
-Here $\mathbf{W}_{xc} \in \mathbb{R}^{d \times h}$ and $\mathbf{W}_{hc} \in \mathbb{R}^{h \times h}$ are weights and $\mathbf{b}_c \in \mathbb{R}^{1 \times h}$ is a bias.
+Here $\mathbf{W}_{xc} \in \mathbb{R}^{d \times h}$ and $\mathbf{W}_{hc} \in \mathbb{R}^{h \times h}$ are weight parameters and $\mathbf{b}_c \in \mathbb{R}^{1 \times h}$ is the bias parameter.
+
+As quick illustration of the candidate memory cell are shown in :numref:`lstm_1`.
```
Comment (astonzhang) [inline chapter_recurrent-neural-networks/lstm.md]: grammatically wrong

File: chapter_recurrent-neural-networks/lstm.md (hunk 1 above)
Comment (goldmermaid) [reply chapter_recurrent-neural-networks/lstm.md]: make sense

File: chapter_recurrent-neural-networks/lstm.md (hunk 3 above)
Comment (goldmermaid) [reply chapter_recurrent-neural-networks/lstm.md]: okay

File: chapter_recurrent-neural-networks/lstm.md (hunk 2 above)
Comment (goldmermaid) [reply chapter_recurrent-neural-networks/lstm.md]: make sense

**Pull request merged**